ga4gh/datamodel/__init__.py

```diff
@@ -508,6 +529,21 @@ def getParentContainer(self):
         """
         return self._parentContainer
 
+    def _scanDataFiles(self, dataDir, patterns):
```

Comment:
@dcolligan it would be great if there was one pattern :). The form of the g2p data is just a single `.ttl` file I think.